database/init.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const _ = require('lodash')
+const async = require('async')
+const pg = require('pg')
+const config = require('./../src/lib/config')
+
+if (config.get('env') !== 'test') {
```

Comment:
I think due to naming I misunderstood what you wanted to do. Yes let's rename it to something else and not use the env term. I'll approve it.

i would imagine a structure along the lines:
- you have in the config folder an config/index.js that contains the config values set from process.env and on top of it it is applied a dev.js config,
- then a config/test.js on which you load the index.js structure and apply on top the dev then test flag property.

And in code you require `./config` or `./config/test` based on needs.